finetune/config.py

```diff
@@ -260,30 +282,39 @@ def get_default_config():
 
         # Regression Params
         regression_loss="L2",
-        prefit_init=False,
 
         # Association Params
         viable_edges=None,
         association_types=None,
         assocation_loss_weight=100.0,
 
-        # Must remain fixed
-        n_heads=12,
-        n_layer=12,
-        act_fn="gelu",
-        n_embed=768,
-        base_model_path="Base_model.jl"
+        # Location of model weights
+        base_model=GPT2Model,
+        base_model_path=finetune_model_path(
+            os.path.join("gpt2", "model-sm.jl")
+        ),
+
+        # Possible `SourceModel` specific settings
+        n_heads=None,
+        n_layer=None,
+        act_fn=None,
+        n_embed=None,
     )
+    return settings
 
 
 def get_small_model_config(**kwargs):
```

Comment:
Instead of having this function, I think a more systematic way to use the small model as the base would be to create a GPTSmallModel in the model.py folder for GPT-1. That way, using the small GPT-1 model is just like using any of the other base models, i.e. calling get_config while specifying the appropriate base_model in the kwargs.

Reply:
Strong agreement, that's older code I should have updated with this PR. I'll make that change when I'm back in Boston later this week! Thanks for the code review!